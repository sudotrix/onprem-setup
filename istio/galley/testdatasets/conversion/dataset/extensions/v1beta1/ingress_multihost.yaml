apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  name: echo
  annotations:
    kubernetes.io/ingress.class: cls
spec:
  rules:
  - host: echo1.example.com
    http:
      paths:
      - backend:
          serviceName: echo1
          servicePort: 80
  - host: echo2.example.com
    http:
      paths:
      - backend:
          serviceName: echo2
          servicePort: 80