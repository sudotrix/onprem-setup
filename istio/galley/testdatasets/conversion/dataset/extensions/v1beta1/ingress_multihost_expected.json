{
    "istio/networking/v1alpha3/gateways": [
        {
            "Metadata": {
                "name": "istio-system/echo-istio-autogenerated-k8s-ingress"
            },
            "Body": {
                "selector": {
                    "istio": "ingress"
                },
                "servers": [
                    {
                        "hosts": [
                            "*"
                        ],
                        "port": {
                            "name": "http-80-i-echo-{{.Namespace}}",
                            "number": 80,
                            "protocol": "HTTP"
                        }
                    }
                ]
            },
            "TypeURL": "type.googleapis.com/istio.networking.v1alpha3.Gateway"
        }
    ],
    "istio/networking/v1alpha3/virtualservices": [
        {
            "Metadata": {
                "name": "istio-system/echo1-example-com-echo-istio-autogenerated-k8s-ingress"
            },
            "Body": {
                "gateways": [
                    "istio-autogenerated-k8s-ingress"
                ],
                "hosts": [
                    "echo1.example.com"
                ],
                "http": [
                    {
                        "match": [
                            {}
                        ],
                        "route": [
                            {
                                "destination": {
                                    "host": "echo1.{{.Namespace}}.svc.cluster.local",
                                    "port": {
                                        "number": 80
                                    }
                                },
                                "weight": 100
                            }
                        ]
                    }
                ]
            },
            "TypeURL": "type.googleapis.com/istio.networking.v1alpha3.VirtualService"
        },
        {
            "Metadata": {
                "name": "istio-system/echo2-example-com-echo-istio-autogenerated-k8s-ingress"
            },
            "Body": {
                "gateways": [
                    "istio-autogenerated-k8s-ingress"
                ],
                "hosts": [
                    "echo2.example.com"
                ],
                "http": [
                    {
                        "match": [
                            {}
                        ],
                        "route": [
                            {
                                "destination": {
                                    "host": "echo2.{{.Namespace}}.svc.cluster.local",
                                    "port": {
                                        "number": 80
                                    }
                                },
                                "weight": 100
                            }
                        ]
                    }
                ]
            },
            "TypeURL": "type.googleapis.com/istio.networking.v1alpha3.VirtualService"
        }
    ]
}